service/create/service.go

```diff
@@ -794,10 +794,53 @@ func (s *Service) processCluster(cluster awstpr.CustomObject) error {
 		return microerror.Maskf(executionFailedError, fmt.Sprintf("could not make subnet public, '%#v'", err))
 	}
 
-	if err := publicSubnet.AddHostVPCRoute(routeTable, vpcPeeringConection.(*awsresources.VPCPeeringConnection)); err != nil {
+	conn, err := vpcPeeringConection.(*awsresources.VPCPeeringConnection).FindExisting()
+	if err != nil {
+		return microerror.Maskf(executionFailedError, fmt.Sprintf("could not find vpc peering connection: '%#v'", err))
+	}
+
+	routeTableID, err := routeTable.GetID()
+	if err != nil {
+		return microerror.Maskf(executionFailedError, fmt.Sprintf("could not get route table ID: '%#v'", err))
+	}
+
+	publicRoute := &awsresources.Route{
+		RouteTableID:         routeTableID,
+		DestinationCidrBlock: *conn.AccepterVpcInfo.CidrBlock,
+		VpcID:                *conn.VpcPeeringConnectionId,
+		AWSEntity:            awsresources.AWSEntity{Clients: clients},
+	}
+
+	if err := publicRoute.CreateOrFail(); err != nil {
 		return microerror.Maskf(executionFailedError, fmt.Sprintf("could not add host vpc route: '%#v'", err))
 	}
 
+	for _, privateSubnetSpec := range cluster.Spec.AWS.VPC.PrivateSubnets {
+
+		privateRouteTable := &awsresources.RouteTable{
+			Name:   privateSubnetSpec.RouteTableName,
+			VpcID:  cluster.Spec.AWS.VPC.PeerID,
+			Client: hostClients.EC2,
+		}
+
+		privateRouteTableID, err := privateRouteTable.GetID()
+		if err != nil {
+			return microerror.Maskf(executionFailedError, fmt.Sprintf("could not get private route table ID: '%#v'", err))
```

Comment:
Might be good if the errors included the route table name so we can see which has failed.